src/hotspot/share/prims/jvmti.xml

```diff
@@ -11225,6 +11231,68 @@ myInit() {
       </errors>
     </function>
 
+    <function id="GetGCCpuTimerInfo" phase="any" callbacksafe="safe" num="157" since="26">
+      <synopsis>Get Total GC Cpu Timer Information</synopsis>
+      <description>
+        Get information about the
+        <functionlink id="GetTotalGCCpuTime"/> timer.
+        The fields of the <datalink id="jvmtiTimerInfo"/> structure
+        are filled in with details about the timer.
+        This information will not change during a particular invocation of the VM.
+      </description>
+      <origin>new</origin>
+      <capabilities>
+        <required id="can_get_gc_cpu_time">
+            Can get GC cpu time.
+        </required>
+      </capabilities>
+      <parameters>
+        <param id="info_ptr">
+          <outptr><struct>jvmtiTimerInfo</struct></outptr>
+          <description>
+            On return, filled with information describing the time
+            returned by <functionlink id="GetTotalGCCpuTime"/>.
+          </description>
+        </param>
+      </parameters>
+      <errors>
+      </errors>
+    </function>
+
+    <function id="GetTotalGCCpuTime" phase="any" callbacksafe="safe" num="158" since="26">
+      <synopsis>Get Total GC CPU Time</synopsis>
+      <description>
+          Return the current value of the total GC CPU timer, in nanoseconds.
+          <p/>
+          The value returned represents nanoseconds since some fixed but
+          arbitrary time (perhaps in the future, so values may be
+          negative).  This function provides nanosecond precision, but not
+          necessarily nanosecond accuracy. No guarantees are made about
+          how frequently values change.
+          <p/>
+          Get information about this timer with
+          <functionlink id="GetGCCpuTimerInfo"/>.
+      </description>
+      <origin>new</origin>
+      <capabilities>
+        <required id="can_get_gc_cpu_time">
+            Can get GC cpu time.
+        </required>
+      </capabilities>
+      <parameters>
+        <param id="nanos_ptr">
+          <outptr><jlong/></outptr>
+          <description>
+            On return, points to the time in nanoseconds.
+            This is an unsigned value.  If tested or printed as a jlong (signed value)
+            it may appear to be a negative number.
```

Comment:
This seems to contradict the description that it could be relative to a value in the future and hence negative - thus it can't be "unsigned". But then I don't see how it can be as described - for this timer to be useful it must be tracking nanoseconds of CPU time consumed by GC since VM startup.